Assignment 2.Rmd

```diff
@@ -87,13 +91,85 @@ plot(g,layout=layout.fruchterman.reingold, vertex.color=VERTEX$gender, edge.widt
 
 In Part II your task is to [look up](http://igraph.org/r/) in the igraph documentation and create a graph that sizes the student vertices in terms of the number of disciplinary actions they have recieved, and the teachers in terms of the number of disciplinary actions they have given out.
 
+```{r}
 
+EDGES.ST <- EDGE %>% group_by(to) %>%summarise(sum(count))
+EDGES.TE <- EDGE %>% group_by(from) %>% summarise(sum(count))
+names(EDGES.ST) <- c("id", "count")
+names(EDGES.TE) <- c("id", "count")
+VERTEX <- bind_rows(EDGES.ST, EDGES.TE)
+g <- graph.data.frame(EDGE, directed=TRUE, vertices=VERTEX)
+plot(g,layout=layout.fruchterman.reingold, vertex.size=VERTEX$count, edge.width=EDGE$count)
```

Comment:
Plot looks great! You may want to adjust the size of the arrow heads though - they are obscuring things a little.